securityhub-org/sumologic-securityhub-collector-awsorg/sam/template.yaml

```diff
@@ -40,7 +40,7 @@ Resources:
     Type: 'AWS::Serverless::Function'
     Properties:
       Handler: securityhub_collector_org.lambda_handler
-      Runtime: python3.7
+      Runtime: python3.11
       CodeUri: ../src/
```

Comment:
Where is this code located? Is it checked-in to some repo and/ or s3 bucket?
If yes can you please share the code location path.
It is recommended to use S3 path in codeUri for future maintenance purposes.